sklearn/preprocessing/data.py

```diff
@@ -229,11 +233,17 @@ class MinMaxScaler(BaseEstimator, TransformerMixin):
 
     scale_ : ndarray, shape (n_features,)
         Per feature relative scaling of the data.
+
+
+
     """
 
-    def __init__(self, feature_range=(0, 1), copy=True):
+    def __init__(self, feature_range=(0, 1), copy=True, selected="all"):
         self.feature_range = feature_range
         self.copy = copy
+        self.selected = selected
+        self.scaled_features_ = None if selected == "all" else selected
```

Comment:
I addressed this problem in my most recent commit here: https://github.com/scikit-learn/scikit-learn/pull/5002/files.

Now, the only logic in the constructor is checking whether the passed in `selected` list has any strings. Should I move that to `fit`?